exporters/otlp/common/src/main/java/io/opentelemetry/exporter/otlp/internal/grpc/DefaultGrpcExporterBuilder.java

```diff
@@ -166,7 +166,11 @@ public DefaultGrpcExporterBuilder(
    *
    * @throws IllegalArgumentException if the instance does not contain a field called "delegate" of
    *     type {@link DefaultGrpcExporterBuilder}
+   * @deprecated Use {@link
+   *     io.opentelemetry.exporter.otlp.internal.retry.RetryUtil#setRetryPolicyOnDelegate(Object,
+   *     RetryPolicy)}
    */
+  @Deprecated
```

Comment:
since this is `internal` can we just delete this?

Reply:
I would like to, yes 👍